runtime/src/iouring/mod.rs

```diff
@@ -17,6 +26,15 @@ pub struct Config {
     pub iopoll: bool,
     /// If true, use single issuer mode.
     pub single_issuer: bool,
+    /// In the io_uring event loop (`run`), wait at most this long for a new
+    /// completion before checking for new work to submit to the io_ring.
+    /// If None, wait indefinitely. In this case, caller must ensure that operations
+    /// submitted to the io_uring complete so that they don't block the event loop.
+    /// For example, do not submit a recv operation and then a write operation
```

Comment:
nit: I'd a \n before this `For example...`

You may also want to include a note that this pattern is common in networking implementations (where we immediately try to recv on the channel)?

Reply:
Made the comment more verbose and added newline